src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -122,6 +122,10 @@ public function compile($path = null)
                 $this->files->get($this->getPath())
             );
 
+            if (! empty($this->getPath())) {
+                $contents .= "<?php /**PATH {$this->getPath()} ENDPATH**/ ?>";
```

Comment:
```suggestion
                $contents .= "<?php #PATH {$this->getPath()} ENDPATH# ?>";
```

Maybe just a slightly different comment syntax for aesthetics and shorter regex pattern?

Reply:
Could be possible yes. Gonna wait what @bzixilu says.